tests/test_arcaflow_plugin_kill_pod.py

```diff
@@ -197,4 +232,4 @@ def remove_test_pod():
 
 
 if __name__ == '__main__':
-    unittest.main()
+    unittest.main(warnings="ignore")
```

Comment:
In [latest run](https://github.com/arcalot/arcaflow-plugin-kill-pod/actions/runs/3387965200/jobs/5637657034) there were a lot of warnings that clog up the output and make it hard to find actual errors, I was hoping that this would help get rid of some of those

Example: 
`ResourceWarning: Enable tracemalloc to get the object allocation traceback`

Reply:
Will add a comment

Reply:
This might be indicative of another problem, see https://stackoverflow.com/questions/60945317/python-selenium-resourcewarning-enable-tracemalloc-to-get-the-object-allocati.